### PR TITLE
3.5 mass piece load configurer update

### PR DESCRIPTION
**Pull request by BrentEaston**

### CODE REVIEW

File: vassal-app/src/main/java/VASSAL/counters/MassPieceLoader.java [hunk 1]
```diff
@@ -602,7 +594,7 @@ public MassPieceDefiner() {
    * template.
    *
    */
-  class DefineDialog extends JDialog {
+  static class DefineDialog extends JDialog {
```
Comment (uckelman) [inline vassal-app/src/main/java/VASSAL/counters/MassPieceLoader.java:597]: Should this be private?

File: vassal-app/src/main/java/VASSAL/counters/MassPieceLoader.java [hunk 2]
```diff
@@ -773,7 +760,7 @@ public TableCellRenderer getCellRenderer(int row, int column) {
   /**
    * Blank Cell Renderer
    */
-  class NullRenderer implements TableCellRenderer {
+  static class NullRenderer implements TableCellRenderer {
```
Comment (uckelman) [inline vassal-app/src/main/java/VASSAL/counters/MassPieceLoader.java:763]: Same here---private?

File: vassal-app/src/main/java/VASSAL/counters/MassPieceLoader.java [hunk 3]
```diff
@@ -785,7 +772,7 @@ public Component getTableCellRendererComponent(JTable arg0, Object arg1,
    * *************************************************************************
    * Custom TreeTable Node
    */
-  class BasicNode extends DefaultMutableTreeTableNode {
+  static class BasicNode extends DefaultMutableTreeTableNode {
```
Comment (uckelman) [inline vassal-app/src/main/java/VASSAL/counters/MassPieceLoader.java:775]: Ditto.

File: vassal-app/src/main/java/VASSAL/counters/MassPieceLoader.java [hunk 4]
```diff
@@ -916,7 +901,7 @@ public void setSkip(boolean skip) {
   /**
    * Node representing a Layer trait of a GamePiece
    */
-  class LayerNode extends BasicNode {
+  static class LayerNode extends BasicNode {
```
Comment (uckelman) [inline vassal-app/src/main/java/VASSAL/counters/MassPieceLoader.java:904]: Ditto.

File: vassal-app/src/main/java/VASSAL/counters/MassPieceLoader.java [hunk 5]
```diff
@@ -936,7 +921,7 @@ public String getName() {
    * Node representing an individual Image Level within a Layer trait
    *
    */
-  class LevelNode extends BasicNode {
+  static class LevelNode extends BasicNode {
```
Comment (uckelman) [inline vassal-app/src/main/java/VASSAL/counters/MassPieceLoader.java:924]: Ditto.

File: vassal-app/src/main/java/VASSAL/counters/MassPieceLoader.java [hunk 6]
```diff
@@ -961,7 +946,7 @@ public String getName() {
    * load flag
    *
    */
-  class PieceInfo {
+  static class PieceInfo {
```
Comment (uckelman) [inline vassal-app/src/main/java/VASSAL/counters/MassPieceLoader.java:949]: Ditto.

File: vassal-app/src/main/java/VASSAL/counters/MassPieceLoader.java (hunk 1 above)
Comment (BrentEaston) [reply vassal-app/src/main/java/VASSAL/counters/MassPieceLoader.java]: Yes indeed. We really don't want people subclassing internal parts of the mass Piece Loader. I have made all the internal classes private

**Pull request merged**